app/ui.js

```diff
@@ -476,7 +476,7 @@ var UI;
                     break;
             }
 
-            statusElem.innerHTML = text;
+            statusElem.innerHTML = Util.Localisation.get(text);
```

Comment:
This is generally not a good idea as it makes it very difficult to find what needs to be translated. Only put translations on literals.